Cargo.toml

```diff
@@ -34,7 +38,7 @@ mimalloc = { version = "0.1.29", default-features = false, optional = true }
 hex = "0.4.3"
 cargo-llvm-cov = "0.5.14"
 anyhow = "1.0.66"
-starknet-contract-class = { path = "crates/starknet-contract-class" }
+starknet-contract-class = "0.1.0"
```

Comment:
This can prevent issues such as this one: https://github.com/lambdaclass/starknet_in_rust/pull/590#discussion_r1218563067. Where code used by another crate was removed without any warning

Reply:
I tried to publish it with a relative path and it asked me to change it to a pinned version

Reply:
In [_cairo-rs_](https://github.com/lambdaclass/cairo-rs/blob/f1ad334b07f1c73f16a4fed7d5db67af62105a41/Cargo.toml#L58) we have both a _path_ and a _version_ specified for the _felt_ package.